packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1067,6 +1067,35 @@ oie.registration.form.submit = Sign Up
 oie.user.profile.lastname= Last name
 oie.user.profile.firstname= First name
 oie.user.profile.primary.email= Email
+oie.user.profile.login = Username
+oie.user.profile.middleName = Middle name
+oie.user.profile.honorificPrefix = Honorific prefix
+oie.user.profile.honorificSuffix = Honorific suffix
+oie.user.profile.primaryEmail = Primary email
```

Comment:
May be conflicting with the entry on `L#1069` which ends in '`.primaryEmail`' but the new addition specifies the type of email in the string (Primary)

> oie.user.profile.primary.email= Email

Reply:
It looks like the keys is based on `oie.user.profile` + variable name in schema. I am not sure if OIE and standard user profile are the same. But in standard user profile, it only have variable name `email` for Primary email. Do we know if primaryEmail will cause any issue?

Reply:
I'm not sure if that would cause an issue, @nikhilvenkatraman-okta would you have any insight on this?

Reply:
I am not sure. This seems like a question for the home team.

Reply:
@alexma-okta I don't think it will cause a problem since we are just using these keys for localization